README.rst

```diff
@@ -42,7 +42,8 @@ Author       National Instruments
 About
 =====
 
-The **nimi-python** repository generates Python bindings (Application Programming Interface) for interacting with the Modular Instrument drivers. Currently, the following drivers are supported:
+The **nimi-python** repository generates Python bindings (Application Programming Interface) for interacting with the Modular Instrument drivers. Currently,
```

Comment:
I'd change the last sentece to say:

"The following driver APIs are supported" (add API, remove Currently)

Reply:
Changed